fe-nc-news-project/src/utils.js

```diff
@@ -0,0 +1,7 @@
+import axios from "axios"
+
```

Comment:
nice job extracting this out into a seperate file, maybe look into creating a baseURL with axios